src/tribler-core/tribler_core/modules/tests/test_versioncheck.py

```diff
@@ -108,30 +109,34 @@ async def test_non_json_response(version_check_manager, version_server):
         await version_check_manager.check_new_version()
 
 
-@pytest.mark.skip
 @pytest.mark.asyncio
 async def test_version_check_timeout(version_check_manager, version_server):
-    #await setup_version_server(json.dumps({'name': 'v1337.0'}))
+    global response
+    response = json.dumps({'name': NEW_VERSION_ID})
+
+    import tribler_core.modules.versioncheck_manager as vcm
+    old_timeout = vcm.VERSION_CHECK_TIMEOUT
+    vcm.VERSION_CHECK_TIMEOUT = 0.001
+
+    has_new_version = await version_check_manager.check_new_version()
+    assert not has_new_version
+
+    vcm.VERSION_CHECK_TIMEOUT = old_timeout
```

Comment:
It seems that there is a test for the timeout of the version check, but the code coverage reporter shows an uncovered line there?

Reply:
Yep, we *definitely* want to cover *that* line.

Reply:
@devos50 @ichorid to address this I have now added a separate test to cover the timeout case for individual API check